lib/Target/SubstraitPB/Import.cpp

```diff
@@ -106,6 +111,130 @@ static mlir::FailureOr<CrossOp> importCrossRel(ImplicitLocOpBuilder builder,
   return builder.create<CrossOp>(leftVal, rightVal);
 }
 
+static mlir::FailureOr<ExpressionOpInterface>
+importExpression(ImplicitLocOpBuilder builder, const Expression &message) {
+  MLIRContext *context = builder.getContext();
+  Location loc = UnknownLoc::get(context);
+
+  Expression::RexTypeCase rex_type = message.rex_type_case();
+  switch (rex_type) {
+  case Expression::RexTypeCase::kSelection: {
+    return importFieldReference(builder, message.selection());
+  }
+  case Expression::RexTypeCase::kLiteral: {
```

Comment:
Lets keep it in alphabetical order unless order matters

Reply:
Done.